src/hub.js

```diff
@@ -1,9 +1,17 @@
-const { startDriver } = require("./driver/handler")
-const { startVendor } = require("./vendor/handler")
-
-startDriver();
-startVendor();
-// setTimeout(()=>{
-//     startVendor();
-// }, 1500)
-console.log("everything is started")
+
+const { Server } = require("socket.io");
+const io = new Server(3333)
+
+// startDriver();
+// startVendor();
+
+
+function startEventServer(){
+    io.on("connection",()=>{
+        console.log("have new connection");
```

Comment:
Here, you'll tell the hub to start or emit the events of 'delivered' and 'pickup'. You can take a look at the demo code in the class repo if you need to. Also, you can break it up and put notes on how it works or explain what is going on in your own words if that will help you understand it.